runtime/compiler/runtime/Runtime.cpp

```diff
@@ -167,6 +168,29 @@ void* TR_RuntimeHelperTable::getFunctionPointer(TR_RuntimeHelper h)
 
 void* TR_RuntimeHelperTable::getFunctionEntryPointOrConst(TR_RuntimeHelper h)
    {
+#if defined(J9VM_OPT_JITSERVER)
+   TR::CompilationInfo *compInfo = getCompilationInfo(jitConfig);
+   auto compInfoPT = TR::compInfoPT;
+   if (compInfo->getPersistentInfo()->getRemoteCompilationMode() == JITServer::SERVER &&
```

Comment:
I would like to limit the overhead for the non-jitserver path, so `auto compInfoPT = TR::compInfoPT;` should be computed after `if (compInfo->getPersistentInfo()->getRemoteCompilationMode() == JITServer::SERVER`
For the JITServer path, is it all possible that `compInfoPT` does not exist? I am guessing not because at JITServer most of the code is executed during a compilation.

Reply:
This method is called at jit initialization, so `compInfoPT` could be NULL.